### ISSUE TITLE
can we use single litmus UI for chaos test in multiple cluster

### ISSUE THREAD

**Issue by rishank69** [litmuschaos/litmus#3345]

**Comment by gdsoumya** [comment]
Yes, Litmus is a cross-cloud control plane, you can connect multiple agents running on different clusters and execute experiments on each of those clusters.

**Comment by rishank69** [comment]
> Yes, Litmus is a cross-cloud control plane, you can connect multiple agents running on different clusters and execute experiments on each of those clusters.

Sure will try with agent installation, i wanted to test in our staging and dev environment. do I need to install in cluster mode or namespace mode would work fine ?

**Comment by gdsoumya** [comment]
Ns mode has some manual steps for the CRD installation, you can follow the guide but in general it should work fine.

**Comment by rishank69** [comment]
> Ns mode has some manual steps for the CRD installation, you can follow the guide but in general it should work fine.

sure, i tried by installing an agent in another cluster but agent are are coming into pending state , is there any way to check logs  , `5f19ceb-08d1-4f58-a68b-fd5236bd99cd    staging         INACTIVE`

**Comment by Rezkmike** [comment]
@

> > Ns mode has some manual steps for the CRD installation, you can follow the guide but in general it should work fine.
> 
> sure, i tried by installing an agent in another cluster but agent are are coming into pending state , is there any way to check logs , `5f19ceb-08d1-4f58-a68b-fd5236bd99cd staging INACTIVE`

is the single UI able to handle multi-cluster?

**Comment by Handsome-Jaxon** [comment]
> > Ns mode has some manual steps for the CRD installation, you can follow the guide but in general it should work fine.
> 
> sure, i tried by installing an agent in another cluster but agent are are coming into pending state , is there any way to check logs , `5f19ceb-08d1-4f58-a68b-fd5236bd99cd staging INACTIVE`

You can check the subscriber's logs, also you can check the configMap `subscriber-config`, make sure your `SERVER_ADDR` point to your chaos server correctly (domain / protocol).

**Comment by DistantThunder** [comment]
Hey, I'm sorry if this is the wrong issue to be asking this but @gdsoumya is it possible to have One experiment running on multiple clusters simultaneously? As in can you pilot multiple clusters as one?
